add czi funding announcement (#346)
open-life-science/open-life-science.github.io: _posts/2022-01-27-runway-to-sustainability-czi-funding.md

---
layout: post
title: "Runway for sustainability: OLS awarded funding from the Chan Zuckerberg Initiative to power its next phase"
authors:
- bebatut
- emmyft
- malvikasharan
- yochannah
image: ../images/2022-01-27-ols-czi-funding.jpg
photos:
  name: Photo by Julian Hochgesang on Unsplash
  url: https://unsplash.com/@julianhochgesang
---

> OLS is absolutely delighted to share the news that as of January 2022, the Chan Zuckerberg Initiative has awarded OLS a total of $574,945.30 USD over the next two years. This grant will provide OLS with a truly valuable opportunity to improve sustainability for the organisation and community, and allow enough time to focus on planning for the longer term.

Reviewer: ```suggestion
OLS is absolutely delighted to share the news that as of January 2022, the Chan Zuckerberg Initiative has awarded OLS a total of USD 574,945.30 over the next two years. This grant will provide OLS with a truly valuable opportunity to improve sustainability for the organisation and community, and allow enough time to focus on planning for the longer term.
```

Activities over 2022 and 2023 will include:

> 1. **Four open research mentoring and training cohorts**: OLS-5 applications have closed, and we will run OLS-6 in the second half of 2022, followed by OLS-7 and OLS-8 in 2023.

Reviewer: ```suggestion
1. **Four open research mentoring and training cohorts**: OLS-5 applications have closed, and we will run OLS-6 in the second half of 2022, followed by OLS-7 and OLS-8 in 2023. As in the last cohorts, OLS will continue to cater to researchers and the open-research curious from all backgrounds.
```

Moving this from the previous point!

Author: I'm glad we retained it! :)

> 2. **Adding an additional optional Open Source Software module to the OLS curriculum**: - OLS will continue to cater to researchers and the open-research curious from all backgrounds, so this module will specifically be a skill-up enhancement for those who have software and code-related projects and wish to attend it.

Reviewer: ```suggestion
2. **New Open Source Software module to the OLS curriculum**: - OLS team will co-design a new module on Open Source Software to be delivered as an additional skill-up enhancement for those who have software and code-related projects and wish to attend it. 
```

> 3. **Sustainability and business model planning**: Until recently, OLS was fully volunteer driven. For cohorts 3 and 4, we were able to offer small payments in the form of honoraria to some of our mentors and call facilitators. This will be a core offering over 2022 and 2023, as well as dedicated time to pay the OLS core team to work on OLS.

Reviewer: ```suggestion
3. **Sustainability and business model planning**: Until recently, OLS was fully volunteer-driven. For cohorts 3 and 4, we were able to offer small payments in the form of honoraria to some of our mentors and call facilitators. This will be a core offering over 2022 and 2023, as well as dedicated time to pay the OLS core team to work on OLS.
```

> In addition to paying mentors, call facilitators, experts, and directors, we intend to use this time to develop a sustainability plan beyond the end of 2023 - not only financial, but community sustainability. Our community is the centre of OLS, and we want to figure out ways to keep it healthy and thriving, serving its members and furthering our mission of open, equitable, and community-driven research.

Reviewer: ```suggestion
In addition to paying mentors, call facilitators, experts, and directors, we intend to use this time to develop a sustainability plan beyond the end of 2023 - not only financial but community sustainability. Our community is the centre of OLS, and we want to figure out ways to keep it healthy and thriving, serving its members and furthering our mission of open, equitable, and community-driven research.
```

While we’ve been a registered company for some time now, we’re having to do a lot of growing up, too: establishing a payroll, learning about international tax and employment laws, meeting with accountants and writing policies.

- **Stepping up: an Executive Director**: We are extremely delighted to announce that Yo Yehudi will join OLS as Executive Director starting from February 1st, to help drive the initiatives in this grant. The remaining members of the OLS Core Team (Emmy, Bérénice, Malvika) will also be recompensed to work on various components of OLS in different capacities.

> - **Establishing collaborations**: A core part of our sustainability planning will be working with and embracing other organisations interested in open research - organisations that might benefit from open research training and mentoring, including tailored inhouse deep-dives. It’s also essential to us that our core experience of OLS remain free and accessible to all, and that we retain focus on qualitative interactions, equity, and a global network.

Reviewer: ```suggestion
- **Establishing collaborations**: A core part of our sustainability planning will be working with and embracing other organisations interested in open research - organisations that might benefit from open research training and mentoring, including tailored in-house deep-dives. It’s also essential to us that our core experience of OLS remains free and accessible to all, and that we retain focus on qualitative interactions, equity, and a global network.
```

**How can you be part of this work?**

## We’d love to work with you over the next 2 years.
- Open-source research tools developers, maintainers and users- let’s co-design a module and curriculum!
- **Governance planning and establishment**: As mentioned in our previous blog post, we aspire to be radically transparent in our governance. With you, we want to build and strengthen diverse pathways to community leadership, to ensure accountability and capacity to continue to serve our and other open communities.
- **Collaborations**: Are you working in research or a related domain, and interested in collaborating, forming institutional partnerships, or hiring our services?

For all of these - please do contact team@openlifesci.org to set up a chat!

## In conclusion:

> We’re delighted to be able to direct paid staff time to looking forwards, rather than merely existing in the day-to-day to run the program, and we wouldn’t have been able to do this without the generous grant from CZI. Huge thanks in particular to Carly Strasser, Dario Taraborelli, and everyone on the CZI Open Science team who recognise the importance of both training and mentorship in open research practices, as well as providing well-needed financial stability to initiatives that previously have primarily been volunteer efforts. And as always - thank you to our community of mentors, experts, project leads, co-facilitators, partners and friends, who have supported and shaped OLS, tolerated our blunders and sustained our growth.

Reviewer: ```suggestion
We’re delighted to be able to direct paid staff time to look forwards, rather than merely existing in the day-to-day to run the program, and we wouldn’t have been able to do this without the generous grant from CZI. Huge thanks in particular to Carly Strasser, Dario Taraborelli, and everyone on the CZI Open Science team who recognise the importance of both training and mentorship in open research practices, as well as providing well-needed financial stability to initiatives that previously have primarily been volunteer efforts. And as always - thank you to our community of mentors, experts, project leads, co-facilitators, partners and friends, who have supported and shaped OLS, tolerated our blunders and sustained our growth.
```

> Our grant application documentation, including budget, is available on Zenodo. Please cite them as: Open Life Science Team. (2022). Open Life Science Proposal for the next phase of funding and support from the Chan Zuckerberg Initiative: 2021 project report. Zenodo. https://doi.org/10.5281/zenodo.5907922

Reviewer: ```suggestion
Our grant application documentation, including a budget plan, is available on Zenodo. Please cite them as: Open Life Science Team. (2022). Open Life Science Proposal for the next phase of funding and support from the Chan Zuckerberg Initiative: 2021 project report. Zenodo. https://doi.org/10.5281/zenodo.5907922
```
